Dockerfile

```diff
@@ -15,6 +15,7 @@ WORKDIR /app
 ENV GOPATH=/tmp/gocode
 ENV GOOS=linux
 ENV GOARCH=amd64
+ENV CGO_ENABLED=0
```

Comment:
This enables the cross compiling so that the libraries are included into the executable (more portable!)